crates/egui/src/widgets/text_edit/text_buffer.rs

```diff
@@ -123,5 +123,5 @@ fn byte_index_from_char_index(s: &str, char_index: usize) -> usize {
             return bi;
         }
     }
-    s.len()
+    s.as_bytes().len()
```

Comment:
This change does nothing because `as_bytes()` for string just reinterprets type `&str` as a type `&[u8]`. This is the same thing

Reply:
> This change does nothing because `as_bytes()` for string just reinterprets type `&str` as a type `&[u8]`. This is the same thing

Whoops, thanks, you're right. I had been working under the assumption `&str::len` to be the number of characters, but it is indeed the byte length.